demo/admin/src/App.tsx

```diff
@@ -57,7 +57,7 @@ declare module "@comet/cms-admin" {
     interface ContentScope extends BaseContentScope {}
 
     export interface PermissionOverrides {
-        app: GQLAppPermission;
+        app: GQLPermission;
```

Comment:
We need to use the combined permission enum here if we remove AppPermission from the schema, which is odd. Should we still keep the orphaned types?

Reply:
> Should we still keep the orphaned types?

yes definitly

Reply:
You're right, removed them completely: f63e9842389dbc7879bc4e612727c0b82840b28c.